charts/ark-cluster/templates/deployment.yaml

```diff
@@ -268,4 +270,4 @@ spec:
         {{- if $.Values.extraVolumes }}
         {{- toYaml $.Values.extraVolumes | nindent 8 }}
         {{- end }}
-{{- end }}
\ No newline at end of file
+{{- end }}
```

Comment:
I don't know if these changes are stomping some of your changes or if this was something I did. I'm fine either way as it appears like it does the same thing.

Reply:
Yes, I agree. I'm fine with defining it explicitly.